src/test/java/com/adyen/WebhookTest.java

```diff
@@ -425,4 +424,260 @@ public void testManagementWebhookMerchantCreatedParsing() {
+                "          {\n" +
+                "            \"type\": \"IDENTITY_VERIFICATION\",\n" +
+                "            \"status\": \"PENDING\",\n" +
+                "            \"summary\": {\n" +
+                "              \"kycCheckCode\": 100,\n" +
+                "              \"kycCheckDescription\": \"KYC check summary description\"\n" +
+                "            },\n" +
+                "            \"requiredFields\": [\n" +
+                "              \"field.missing\"\n" +
+                "            ]\n" +
+                "          }\n" +
+                "        ]\n" +
+                "      },\n" +
+                "      \"shareholders\": [\n" +
+                "        {\n" +
+                "          \"checks\": [\n" +
+                "            {\n" +
+                "              \"type\": \"IDENTITY_VERIFICATION\",\n" +
+                "              \"status\": \"PENDING\",\n" +
+                "              \"summary\": {\n" +
+                "                \"kycCheckCode\": 100,\n" +
+                "                \"kycCheckDescription\": \"KYC check summary description\"\n" +
+                "              },\n" +
+                "              \"requiredFields\": [\n" +
+                "                \"field.missing\"\n" +
+                "              ]\n" +
+                "            }\n" +
+                "          ],\n" +
+                "          \"shareholderCode\": \"SH000001\"\n" +
+                "        }\n" +
+                "      ],\n" +
+                "      \"bankAccounts\": [\n" +
+                "        {\n" +
+                "          \"checks\": [\n" +
+                "            {\n" +
+                "              \"type\": \"IDENTITY_VERIFICATION\",\n" +
+                "              \"status\": \"PENDING\",\n" +
+                "              \"summary\": {\n" +
+                "                \"kycCheckCode\": 100,\n" +
+                "                \"kycCheckDescription\": \"KYC check summary description\"\n" +
+                "              },\n" +
+                "              \"requiredFields\": [\n" +
+                "                \"field.missing\"\n" +
+                "              ]\n" +
+                "            }\n" +
+                "          ],\n" +
+                "          \"bankAccountUUID\": \"00000000-0000-0000-0000-000000000000\"\n" +
+                "        }\n" +
+                "      ]\n" +
+                "    }\n" +
+                "  }\n" +
+                "}";
+        ClassicPlatformWebhookHandler webhookHandler = new ClassicPlatformWebhookHandler(notification);
+        Assert.assertTrue(webhookHandler.getAccountHolderCreateNotification().isPresent());
+        AccountHolderCreateNotification payload = webhookHandler.getAccountHolderCreateNotification().get();
+        Assert.assertEquals(payload.getEventType(), "ACCOUNT_HOLDER_CREATED");
```

Comment:
```suggestion
        Assert.assertEquals("ACCOUNT_HOLDER_CREATED", payload.getEventType());
```
It would be nice if we had constants for those event types, after all each *Notification class corresponds to a single eventType?